lib/auth/login.dart

```diff
@@ -44,69 +52,114 @@ class _LoginPageState extends State<LoginPage> {
     });
 
     if (_authResult.ok) {
-      // TODO set global context
-      // Provide user details to next screen
-      Navigator.pushNamed(context, '/home',
-          arguments:
-              AuthDetails(username: user.username, password: user.password));
+      // TODO set global udContext
+      // use popAndPush to stop user pressing back to get to login screen
+      Navigator.popAndPushNamed(
+        context,
+        '/home',
+      );
     } else {
       _authResult.setErrorMessage();
     }
   }
 
+  _launchUrl() async {
+    const url = Constants.url_udStreamRegister;
+    if (await canLaunch(url)) {
+      await launch(url);
+    } else {
+      throw 'Could not launc $url';
+    }
+  }
+
+  _updateCertVal(bool newVal) {
+    setState(() {
+      _ignoreCert = !_ignoreCert;
+    });
+  }
+
   @override
   Widget build(BuildContext context) {
+    // Prevent layout from rotating on this screen
+    SystemChrome.setPreferredOrientations([
+      DeviceOrientation.portraitUp,
+      DeviceOrientation.portraitDown,
+    ]);
     return Scaffold(
       appBar: AppBar(
         title: const Text('Login'),
       ),
-      body: Center(
-        child: Form(
-          key: _formKey,
-          child: Column(
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: <Widget>[
-              TextFormField(
-                decoration: const InputDecoration(
-                  hintText: 'Email or Username',
-                ),
-                validator: (value) {
-                  return value.isEmpty ? 'Please enter your username' : null;
-                },
-                onSaved: (String value) {
-                  user.username = value;
+      body: Form(
+        key: _formKey,
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: <Widget>[
+            KeyboardVisibilityBuilder(
+                builder: (context, child, isKeyboardVisible) {
+                  return isKeyboardVisible
+                      ? emptyWidget
+                      : Image.asset(Constants.img_pointfluentLogo500,
+                          height: 300, width: 300);
                 },
+                child: Image.asset(Constants.img_pointfluentLogo500)),
+            TextFormField(
+              decoration: const InputDecoration(
+                hintText: 'Email or Username',
```

Comment:
```suggestion
                hintText: 'Email',
```
since "Login now uses email instead of username" in udSDK.
Might be worth changing AuthDetails.username to AuthDetails.email as well?